modules/vulnerability-management20-send-reports.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-send-reports_{context}"]
+= Sending vulnerability reports on-demand
+
+[role="_abstract"]
+You can send vulnerability reports immediately, rather than waiting for the scheduled send time.
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Vulnerability Management 2.0* -> *Vulnerability Reporting* and, in the list of report configurations, locate the report configuration for the report that you want to send.
+. Click {kebab}, and then select *Send report now*.
```

Comment:
Most other instances you have elsewhere that are similar to this are in the form:
```
. Click {kebab}. Then, [...]
```
Maybe sync this one up w/ that style as well for consistency.